actionpack/test/dispatch/cookies_test.rb

```diff
@@ -1300,6 +1300,22 @@ def test_vanilla_cookie_with_expires_set_relatively
     end
   end
 
+  def test_purpose_metadata_for_signed_cookies
+   cookies.signed[:user_id] = 50
+   cookies.signed[:discount_percentage] = 10
+
+   cookies[:discount] = cookies[:user_id]
+   assert_nil cookies.signed[:discount_percentage]
```

Comment:
Shouldn't this be `assert_nil cookies.signed[:discount]` instead?

Reply:
@Edouard-chin Yeah, my bad :sweat_smile: